cloud-compose/templates/docker-compose.override.yml

```diff
@@ -1,35 +1,101 @@
-mongos:
-  image: washpost/mongos:3.2
-  command: --configdb="${INSTANCE_IP}:27019"
-  restart: always
-  extra_hosts:
-  {%- for node in aws.nodes %}
-    - "node{{node.id}}:{{node.ip}}" 
-  {%- endfor %}
-  environment:
-    MONGODB_SHARD: node0 
-    MONGODB_HOST: ${INSTANCE_IP}
-    CONFIGDB_HOST: ${INSTANCE_IP}
-    NODE_ID: ${NODE_ID}
-mongodb:
-  image: washpost/mongodb:3.2
-  extra_hosts:
-  {%- for node in aws.nodes %}
-    - "node{{node.id}}:{{node.ip}}" 
-  {%- endfor %}
-  volumes:
-    - "/data/mongodb/mongodb:/data/db"
-  restart: always
-  environment:
-    NODE_ID: ${NODE_ID} 
-    NODE_LIST: node0,node1,node2
-configdb:
-  image: washpost/mongodb:3.2
-  restart: always
-  extra_hosts:
-  {%- for node in aws.nodes %}
-    - "node{{node.id}}:{{node.ip}}" 
-  {%- endfor %}
-  environment:
-    NODE_ID: ${NODE_ID} 
-    NODE_LIST: node0,node1,node2
+version: "2"
+services:
+  mongos:
+    image: washpost/mongos:3.2
+    command: --configdb="${INSTANCE_IP}:27019"
+    restart: always
+    extra_hosts:
+    {%- for node in aws.nodes %}
+      - "node{{node.id}}:{{node.ip}}" 
+    {%- endfor %}
+    {%- if logging is defined and logging.driver is defined and logging.driver == "awslogs" %}
```

Comment:
I think we can be a little more extensible with this by letting the template iterate over the driver:

```yaml
    {% if logging %}
    logging:
      driver: {{ logging.driver }}
      options:
        {% for key, option in logging.options %}
        {{ key }}: {{ option }}
        {% endfor %}
    {% endif %}
```